views/stylesheet.css

```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
+  top:0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+#display {
+  width: 232px;
+  height: 75px;
+  background-color: #8C8C8C;
+}
+
+.calculator-button {
+  display:inline;
+  width: 232px;
+}
+
+.orange-button {
```

Comment:
These styles should be namespaced as well: `.calculator-button-orange`.  Note that I inverted the order as well, to remain consistent with the other elements in this namespace (i.e. to show that this color applied to `.calculator-button`s).